tests/Issues/Issue4499/DoNotDuplicateCode/DoNotDuplicateCodeTest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+declare(strict_types=1);
+
+namespace Rector\Core\Tests\Issues\Issue4499\DoNotDuplicateCode;
+
+use Iterator;
+use Rector\Testing\PHPUnit\AbstractRectorTestCase;
+use Symplify\SmartFileSystem\SmartFileInfo;
+
+final class DoNotDuplicateCodeTest extends AbstractRectorTestCase
+{
+    /**
+     * @dataProvider provideData()
+     */
+    public function test(SmartFileInfo $fileInfo): void
+    {
+        @$this->doTestFileInfo($fileInfo);
```

Comment:
this `@` is on purpose to avoid phpunit break on warning : 

```php
assert(): assert($itemStartPos >= 0 && $itemEndPos >= 0 && $itemStartPos >= $pos)
```

that came from `vendor/nikic/php-parser/lib/PhpParser/PrettyPrinterAbstract.php` on line 759